examples/patterns/navigation/light.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Navigation / Fixed width
+category: _patterns
+---
+
+<header id="navigation" class="p-navigation--light" role="banner">
+  <div class="row">
+    <div class="p-navigation__logo">
+      <a class="p-navigation__link" href="#">
+        Vanilla
+      </a>
+    </div>
+    <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
+    <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Menu</a>
```

Comment:
- For a11y both links should be a button element not a link. We may need to rethink how the toggle works in the future? 

- Requires aria tags as well for elements it controls and visibility.